src/components/Camera/index.js

```diff
@@ -55,7 +56,7 @@ const CameraPure = ({
   isButtonDisabled,
   hasGrantedPermission
 }: Props) => (
-  <div className={classNames(style.camera, className)}>
+  <div className={classNames(style.camera, style[styleClassName], className)}>
```

Comment:
What if `docAutoCaptureFrame` style would sit within Photo component? Then you would not need this additional prop, and could do like such `<Camera className={style.docAutoCaptureFrame} />`.

Reply:
Will try that out. I much prefer that approach actually. Was not sure if the class & style will still get applied when they are in different files/folders.